src/react.force.net.js

```diff
@@ -43,17 +43,17 @@ export const setApiVersion = version => {
 export const getApiVersion = () => apiVersion;
 
 
-/** 
+/**
  * Send arbitray force.com request
  */
-export const sendRequest = (endPoint, path, successCB, errorCB, method, payload, headerParams, fileParams, returnBinary) => {
+export const sendRequest = (endPoint, path, successCB, errorCB, method, payload, headerParams, fileParams, returnBinary, requiresAuthentication) => {
     method = method || "GET";
     payload = payload || {};
     headerParams = headerParams || {};
     fileParams = fileParams || {}; // File params expected to be of the form: {<fileParamNameInPost>: {fileMimeType:<someMimeType>, fileUrl:<fileUrl>, fileName:<fileNameForPost>}}
-    returnBinary = !!returnBinary; // when true response returned as {encodedBody:"base64-encoded-response", contentType:"content-type"}
-
-    const args = {endPoint, path, method, queryParams:payload, headerParams, fileParams, returnBinary};    
+    returnBinary = !!returnBinary;
+    requiresAuthentication = (requiresAuthentication==null)?true:!!requiresAuthentication; // set default to true if missing arg
```

Comment:
Null and undefined should both mean the same thing (false).
So instead of `requiresAuthentication`, the new parameter for `sendRequest` should be the opposite (like `notAuthenticated`). 
I would also send notAuthenticated through the bridge to be backwards compatible.
If someone uses an older `net.react.js`, then nil would be returned and we want that to mean NO.

so I would do
```javascript
export const sendRequest = (endPoint, path, successCB, errorCB, method, payload, headerParams, fileParams, returnBinary, notAuthenticated) => {
    notAuthenticated = !!notAuthenticated; 
    const args = {endPoint, path, method, queryParams:payload, headerParams, fileParams, returnBinary, notAuthenticated};
}
```